python-packages/contract_wrappers/setup.py

```diff
@@ -128,11 +105,27 @@ def run(self):
         rmtree(".mypy_cache", ignore_errors=True)
         rmtree(".tox", ignore_errors=True)
         rmtree(".pytest_cache", ignore_errors=True)
-        rmtree("src/0x_contract_wrappers.egg-info", ignore_errors=True)
+        rmtree(
+            path.join("src", "0x_contract_wrappers.egg-info"),
+            ignore_errors=True,
+        )
         # generated files:
-        for contract in CONTRACTS_TO_BE_WRAPPED:
+        print("Removing src/zero_ex/contract_wrappers/*/__init__.py...")
+        for contract in glob(
+            path.join(
+                ".", "src", "zero_ex", "contract_wrappers", "*", "__init__.py"
+            )
+        ):
             try:
-                remove(f"src/zero_ex/contract_wrappers/{contract}/__init__.py")
+                remove(
+                    path.join(
+                        "src",
+                        "zero_ex",
+                        "contract_wrappers",
+                        f"{contract}",
```

Comment:
why do you have this f-string? can't you simply use the variable `contract`?

Reply:
Good call.  This used to be a `/`-delimited path, and i converted it to `path.join()`.  The old string needed the `f`, but now I see what you mean.  Will update.